flight/Modules/Autotune/autotune.c

```diff
@@ -163,6 +228,8 @@ static void UpdateStabilizationDesired(bool doingIdent) {
 	StabilizationDesiredSet(&stabDesired);
 }
 
+#define MAX_PTS_PER_CYCLE 4
```

Comment:
Ideally, this should depend on the gyro rate, i.e., the module can currently handle 2kHz, but wouldn't work correctly at 3kHz. Correct? Maybe it would make sense for the gyro driver to export the gyro rate used.